routes/secretWord.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const router = express.Router();
+
+
+router.get("/",  (req, res) => {
+  if (!req.session.secretWord) {
+    req.session.secretWord = "syzygy";
+  }
+
+    res.locals.info = req.flash('info') || [];
+    res.locals.errors = req.flash('error') || [];
+    res.render('secretWord', { secretWord: req.session.secretWord });
```

Comment:
You need to pass the variable user to the secretWord page, because it's required by header.ejs file.
Here is how you can do this:
```
res.render('secretWord', { secretWord: req.session.secretWord, user: req.user });
```